packages/core/test/alert/alertTests.tsx

```diff
@@ -35,6 +35,22 @@ describe("<Alert>", () => {
         assert.lengthOf(wrapper.find(`.${Classes.ALERT_FOOTER}`), 1);
     });
 
+    it("renders contents to specified container correctly", () => {
+        const container = document.createElement("div");
+        document.body.appendChild(container);
+        mount(
+            <Alert isOpen={true} container={container}>
+                <p>Are you sure you want to delete this file?</p>
+                <p>There is no going back.</p>
+            </Alert>,
+        );
+        assert.lengthOf(container.getElementsByClassName(Classes.ALERT), 1);
```

Comment:
this is enough here since the other classes are tested by a different test.

Reply:
Tests has been simplified.